qiskit_ibm_runtime/utils/backend_converter.py

```diff
@@ -26,19 +26,16 @@
     WhileLoopOp,
 )
 from qiskit.circuit.gate import Gate
-from qiskit.circuit.library.standard_gates import (
-    get_standard_gate_name_mapping,
-    RZGate,
-    U1Gate,
-    PhaseGate,
-)
-from qiskit.circuit.delay import Delay
+from qiskit.circuit.library.standard_gates import get_standard_gate_name_mapping
 from qiskit.circuit.parameter import Parameter
 from qiskit.providers.backend import QubitProperties
 from qiskit.providers.exceptions import BackendPropertyError
 from qiskit.transpiler.target import InstructionProperties, Target
 
 from ..models import BackendConfiguration, BackendProperties, PulseDefaults
+# is_fractional_gate used to be defined in this module and might be referenced
+# from here externally
+from .utils import is_fractional_gate  # See comment above before removing
```

Comment:
I guess that you'll have to disable the linter's unused import error.

Reply:
The function is still used in `convert_to_target` below. It was defined at the bottom of this file, but I wanted to use it in backend_decoder.py, so I moved it to a more standard location in utils.py alongside similar functions like `are_circuits_dynamic` and `is_isa_circuit`. There had been mention about trying to centralize on `is_fractional_gate` as the central function to use for testing if a gate is fractional so I was worried other code could already be using this location to find the function.